src/JitInterface/src/CorInfoTypes.cs

```diff
@@ -137,9 +137,15 @@ public enum CorTokenType
         mdtBaseType = 0x72000000,
     }
 
-    public enum SIZE_T : ulong { } // Really should be IntPtr
-
-    public enum GSCookie : ulong { } // Really should be IntPtr
+    #if (ARM || ARMEL || X86)
```

Comment:
We do not want to have ifdefs in the CoreRT compiler. Could you please instead replace all uses of this IntPtr.

To preserve the strong typing, GSCookie can be a thin wrapper around IntPtr , like:

```
struct GSCookie
{
     IntPtr _value;
}
```